### PR TITLE
MusicStore: React to pinpoint PR

### PR DESCRIPTION
**Pull request by HaoK**
@Tratcher @PinpointTownes

### CODE REVIEW

**Comment by PinpointTownes** [comment]
Ah great, I didn't know MusicStore used the OAuth2 generic middleware :smile: 

LGTM :+1:

File: test/E2ETests/compiler/shared/Mocks/StartupSocialTesting.cs [hunk 1]
```diff
@@ -103,7 +104,7 @@ public void ConfigureServices(IServiceCollection services)
                 options.ClientId = "[ClientId]";
                 options.ClientSecret = "[ClientSecret]";
                 options.AccessType = "offline";
-                options.Notifications = new GoogleAuthenticationNotifications()
+                options.Notifications = new OAuthAuthenticationNotifications()
```
Comment (PinpointTownes) [inline test/E2ETests/compiler/shared/Mocks/StartupSocialTesting.cs]: `MicrosoftAccountAuthenticationNotifications` was not used in this file?

Comment (PinpointTownes) [inline test/E2ETests/compiler/shared/Mocks/StartupSocialTesting.cs]: Ah, looks like it is: https://github.com/aspnet/MusicStore/blob/haok/7-1pin/test/E2ETests/compiler/shared/Mocks/StartupSocialTesting.cs#L139

Comment (HaoK) [inline test/E2ETests/compiler/shared/Mocks/StartupSocialTesting.cs]: I forgot to save this file, yeah I removed the usings

Comment (PinpointTownes) [inline test/E2ETests/compiler/shared/Mocks/StartupSocialTesting.cs]: Great, thanks :smile:

**Pull request closed without merging**